content/Software Support/Installation/Uninstall-Arduino-IDE.md

```diff
@@ -22,17 +22,64 @@ Learn how to uninstall Arduino IDE.
 
-[The Sketchbook](https://support.arduino.cc/hc/en-us/articles/4412950938514) contains your saved sketches and installed libraries. It is located in your Documents folder by default, inside the `Arduino` subfolder.
+**If you're reinstalling Arduino IDE as a troubleshooting procedure, try the following:**
+
+* To resolve console errors that appear when compiling or uploading sketches, remove the [Arduino15](#Arduino15) and [libraries subfolder](#sketchbook) and restart Arduino IDE.
+* To resolve problems with Arduino IDE itself, remove the [User data folder](#user-data-folder) and restart Arduino IDE.
+
+<a id="sketchbook"></a>
+
+### Sketchbook
+
+The [Sketchbook](https://support.arduino.cc/hc/en-us/articles/4412950938514) contains your saved sketches and installed libraries.
+
+Default Sketchbook location:
 
 * **Windows:** `C:\Users\{username}\Documents\Arduino`
 * **macOS:** `/Users/{username}/Documents/Arduino`
 * **Linux:** `/home/{username}/Arduino`
 
-[The Arduino15 folder](https://support.arduino.cc/hc/en-us/articles/360018448279). Located in these locations by default:
+Default libraries subfolder location:
+
+* **Windows:** `C:\Users\{username}\Documents\Arduino\libraries`
+* **macOS:** `/Users/{username}/Documents/Arduino/libraries`
+* **Linux:** `/home/{username}/Arduino/libraries`
+
+<a id="Arduino15"></a>
+
+### Arduino15 folder
+
+The [Arduino15 folder](https://support.arduino.cc/hc/en-us/articles/360018448279) contain installed board packages and tools.
+
+Default location:
 
 * **Windows:** `C:\Users\{username}\AppData\Local\Arduino15`. Open your Users folder in Explorer and look for a folder called `AppData`.
   * This folder is hidden by default. In newer versions of Windows, you can display hidden items by clicking the **View** tab, and checking _Hidden items_.
 * **macOS:** `/Users/{username}/Library/Arduino15` Open Finder and navigate to the home folder `(/Users/{username})`. From the Finder menu bar, click on `View > Show View Options`, and tick _Show Library Folder_. The `Arduino15` folder can now be found in `{username}/Library`.
 * **Linux:** The folder is located in `home/{username}/.arduino15` and is hidden by default. To show it, click the hamburger button and check Show Hidden Files.
+
+### Configuration folder (.arduinoIDE)
+
+The **.arduinoIDE** folder contain configuration files like `settings.json` and `arduino-cli.yaml`.
+
+Default location:
+
+* **Windows:** `C:\Users\{username}\.arduinoIDE` (hidden)
+* **macOS:** `/Users/{username}/.arduinoIDE` (hidden)
+* **Linux:** `/home/{username}/.arduinoIDE` (hidden)
+
+<a id="user-data-folder"></a>
+
+### User data folder (arduino-ide)
+
+The **arduino-ide** folder contains various temporary files. Removing this folder can sometimes resolve problems with Arduino IDE.
+
+Default location:
+
+* **Windows:** `C:\Users\{username}\AppData\Roaming\arduino-ide\`
+* **macOS:** `~/Library/Application Support/arduino-ide/`
+* **Linux:** `~/.config/arduino-ide/`
+
```

Comment:
```suggestion

This folder is only used by Arduino IDE 2.

```
I would consider adding this information to match the changes in [Find sketches, libraries, board cores, and other files on your computer](https://support.arduino.cc/hc/en-us/articles/4415103213714-Find-sketches-libraries-board-cores-and-other-files-on-your-computer)